src/quote/Quote.ts

```diff
@@ -19,6 +19,16 @@ import Identity from '../identity/Identity'
 import { IQuote, IQuoteAgreement, IQuoteAttesterSigned } from '../types/Quote'
 import { hashObjectAsStr, verify } from '../crypto/Crypto'
 
+/**
+ * Validates the quote against the meta schema and quote data against the provided schema.
+ *
+ * @param schema A [[Quote]] schema object.
+ * @param validate [[Quote]] data to be validated against the provided schema.
+ * @param messages The validating function provides an array of errors, in the case of validation failures.
```

Comment:
Suggestion:
Rephrase line 27 to make the sentence/structure consistent with other parameter doc, e.g.:
"messages: Errors provided by..."

Also:
Out-of-scope for this PR, but validate could/should be replaced by a noun such as dataToValidate

Reply:
Done

Reply:
Not really ☺️ Can you please check again?

Reply:
Not really ☺️ 
Can you please check again?

Reply:
Maybe, I am a little confused.  Because it is what the `AJV` states - https://github.com/epoberezkin/ajv#validation-errors

I have updated and tried again.